assets/model_monitoring/components/src/shared_utilities/histogram_utils.py

```diff
@@ -43,6 +43,12 @@ def get_dual_histogram_bin_edges(
 
         bin_width = (max_value - min_value) / num_bins
 
+        # If histogram has only one value then we only need a single bucket and
+        # should skip the for-loop below.
+        if min_value == max_value:
+            all_bin_edges[col] = [min_value, min_value * 1.01]
```

Comment:
Another problematic case is when `min_value` is a negative number. For these cases, we need to modify the bin edge assignment, so that the smaller value is assigned first. Maybe the easiest approach that covers all scenarios I mentioned earlier is to define `delta = 0.005 if min_value == 0 else abs(min_value * 0.005)`. Please double-check my logic and let me know if something does not make sense.

Reply:
I was just considering this case, I think your solution makes sense.